Probesys/bileto
# new: List users authorizations

Related to https://github.com/Probesys/bileto/issues/88

Changes proposed in this pull request:

- Redirect to /organizations after the creation of an organization
- Show "(yourself)" next to the connected user in the users list
- Display a crown next to the "super admin" role
- Change how the name and description of the "super admin" role are displayed/translated
- Create a sorter of authorizations
- Add an endpoint and a template to list the authorizations of a user

How to test the feature manually:

1. Go to the list of users
2. Display the authorizations of a user
3. Check they are correctly displayed

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
